src/assets/svgs/Logo.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Svg, { Path, Circle } from "react-native-svg";
+const SensumLogo = ({
+  style = null,
+  // scale = 1,
```

Comment:
Why is the scale commented out? IS it necessary to push this line?

Reply:
Branch's still a WIP and svgs doesn't _auto_ so I don't know yet if it'll come in handy for another view or if I'll end up deleting it at the end.

Reply:
oops, didn't see the WIP sign. Sorry.